bin/start_tests.sh

```diff
@@ -11,7 +11,21 @@ deactivate
 # Move down to the dcoscli package
 cd cli
 
+# overwrite core.dcos_url in the dcos.toml
+cp tests/data/dcos.toml $DCOS_CONFIG
```

Comment:
I changed the TC build to make `DCOS_CONFIG` point to `/tmp/teamcity_dcos.toml`. What I think we are doing is coping that file there and running `sed` against `/tmp/teamcity_dcos.toml`. I looked at the content of those files before the tests ran and it looked correct. I couldn't look a the content of those files after the test ran because the build was stuck. Running the build again now.